promisasu/Portal
# Us#82

**Taiga User Story**
TG-82

**Taiga Task(s)**
TG-119 

**What did you change?**
Introduced the retrieval of the data of all the patients that belong into the particular trial into CSV.
Changed files:
1. Added controller/handler/trial-csv.js
2. view/template/trial.hbs - Added a new Button to retrieve the data into CSV.
3. controller/router.js - Added a new route to get the trial id and forward to trial-csv.

**How can we test?**
In the Main Dashboard file, goto any trial. You will see a new Button called "Export Trial Data". Click on that button and you should see a csv file getting downloaded. This file contains the answers to the questions each patient answered.
